pkg/system/system.go

```diff
@@ -433,7 +436,7 @@ func WaitReady() bool {
 			return false, fmt.Errorf("SystemPhaseRejected")
 		}
 		if sys.Status.Phase != nbv1.SystemPhaseReady {
-			return CheckWaitingFor(sys)
+			return false, CheckWaitingFor(sys)
```

Comment:
@jackyalbo Was this a security review comment?

Reply:
@guymguym Not a security issue, just wanted to return once the false,err instead of many times in the function.